.github/workflows/windows.yml

```diff
@@ -100,3 +100,76 @@ jobs:
     - name: test
       working-directory: _build
       run: nmake test VERBOSE_FAILURE=yes TESTS=-test_fuzz* HARNESS_JOBS=4
+  zstd:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ilammy/msvc-dev-cmd@v1
+    - name: prepare the build directory
+      run: mkdir _build
+    - name: Get vcpkg
+      uses: actions/checkout@v2
+      with:
+        path: _build/vcpkg
+        repository: Microsoft/vcpkg
+        # This is the latest tag for vcpkg, it retreives zstd 1.4.4#3
+        # Tip of master retreives zstd 1.4.9
+        ref: 2022.02.02
+    - name: Intialize vcpkg
+      working-directory: _build/vcpkg
+      run: |
+        bootstrap-vcpkg.bat
+        vcpkg integrate install
+    - name: Get zstd
+      working-directory: _build/vcpkg
+      run: |
+        vcpkg install zstd:x64-windows
+    - name: config
+      working-directory: _build
+      run: |
+        perl ..\Configure enable-comp enable-zstd --with-zstd-include=vcpkg\packages\zstd_x64-windows\include --with-zstd-lib=vcpkg\packages\zstd_x64-windows\lib\zstd.lib no-makedepend no-deprecated no-fips no-asm -DOPENSSL_SMALL_FOOTPRINT VC-WIN64A
```

Comment:
Why this: `no-makedepend no-deprecated no-fips no-asm -DOPENSSL_SMALL_FOOTPRINT` (no-fips is the default anyway BTW)